### ISSUE TITLE
Provide list of desktop clients

### ISSUE THREAD

**Issue by bugzy** [nextcloud/notes#724]
Problem:
While it is easy to locate the mobile clients for the notes app, it is pretty difficult to find desktop clients or note taking apps that have some integration with the nextcloud notes app.

Solution:
Provide a document as a readme file in the Repo or as a wiki page that lists desktop editors that provide some integration with the notes app (I know that any standard text editor will work but few provide true integration)

Some desktop applications in the category needed are:
* QOwnNotes -- https://www.qownnotes.org/
* AlephNote -- https://github.com/Mikescher/AlephNote
* MyCloudNotes -- https://github.com/pullus-org/MyCloudNotes
* Nexcloud Notes Mac Client -- https://github.com/dgmid/nextcloud-notes-mac-client

**Comment by stefan-niedermann** [comment]
Makes absolutely sense in my opinion.

@korelstar if you agree, i will spend the [wiki](https://github.com/nextcloud/notes/wiki) a overhaul, categorize the clients by platform and link the here mentioned ones.

I think QOwnNotes is some special case as it does not use the Notes API as far as i know, but it is kind of the most popular Notes editor according to the help forum and twitter... I would keep it here.

**Comment by korelstar** [comment]
Wow, I wasn't aware of the most of those clients.

> @korelstar if you agree, i will spend the [wiki](https://github.com/nextcloud/notes/wiki) a overhaul, categorize the clients by platform and link the here mentioned ones.

That would be great! But some of them haven't been updated for years, so they should be somehow marked as unmaintained.

Maybe you could also remove the old v0.2 API description? Nobody should implement clients for this version anymore.

> I think QOwnNotes is some special case as it does not use the Notes API as far as i know, but it is kind of the most popular Notes editor according to the help forum and twitter... I would keep it here.

Agreed.

**Comment by stefan-niedermann** [comment]
The following comments is a proposal - feedback is welcome :slightly_smiling_face:

**Comment by stefan-niedermann** [comment]
- [3rd-party clients](#3rd-party-clients)
- [3rd-party developer documentation](3rd-party-developer-documentation)

# 3rd-party clients

## Android

- [Nextcloud Notes](https://github.com/stefan-niedermann/nextcloud-notes)
  - [Google Play](https://play.google.com/store/apps/details?id=it.niedermann.owncloud.notes)
  - [F-Droid](https://f-droid.org/repository/browse/?fdid=it.niedermann.owncloud.notes)
- [MyOwnNotes](https://github.com/aykit/MyOwnNotes) (currently not maintained)
  - [Google Play](https://play.google.com/store/apps/details?id=org.aykit.MyOwnNotes)
  - [F-Droid](https://f-droid.org/repository/browse/?fdid=org.aykit.MyOwnNotes)

## iOS

- [CloudNotes](https://github.com/owncloud/notes-iOS-App)
  - [iTunes](https://itunes.apple.com/us/app/cloudnotes-owncloud-notes/id813973264?ls=1&mt=8)


## OS X

- [CloudNotes](https://github.com/owncloud/notes-OSX-App)
  - [Binaries](http://peterandlinda.com/download/CloudNotes.zip) 
- [Nextcloud Notes Client](https://github.com/dgmid/nextcloud-notes-mac-client)
  - [Binaries](https://github.com/dgmid/nextcloud-notes-mac-client/releases/tag/v1.4.4)
- [MyCloudNotes](https://github.com/pullus-org/MyCloudNotes) (currently not maintained)
  - [Build manual](https://github.com/pullus-org/MyCloudNotes#sources-and-build)

## Windows, Linux & OS X

- [QOwnNotes](https://www.qownnotes.org/)
  - [Installation manual](https://www.qownnotes.org/installation/)

## Windows
- [AlephNote](https://github.com/Mikescher/AlephNote)
  - [Installation manual](https://github.com/Mikescher/AlephNote#installation)


# 3rd-party developer documentation

Documentation for people who want to integrate their apps with Nextcloud Notes

* [REST API documentation](https://github.com/nextcloud/notes/blob/master/docs/api/README.md)

**Comment by korelstar** [comment]
Looks fine for me, thank you! Let's go forward with this. :-)

**Comment by stefan-niedermann** [comment]
Done ✅ 

Guess we can simply delete the `0.2` pages of the wiki?

**Comment by bugzy** [comment]
Thanks for the swift action on this. While we are at it here are a few more (feel free to add or not):
https://github.com/scharel/harbour-nextcloudnotes -- for Sailfish OS (mobile)
https://github.com/mathiasconradt/nextcloud-notes-alexa-skill -- Alexa client (unmaintained)
https://github.com/matthoug/gkeep_to_nextcloud_notes -- Google Keep to nextcloud notes migration script
https://github.com/einalex/note -- clipper tool for nc notes

**Comment by stefan-niedermann** [comment]
Well, not sure about the last ones.

Maybe we should further split it up into something like
- `3rd party apps which use the Notes API` and
- `3rd party apps which are compatible with Notes¹`

And add an explanation to the latter, something like
> ¹ 3rd party apps don't need to interact with the Notes API directly. Instead they can just write `.md` or `.txt` files and synchronize them with the notes folder of your Nextcloud. Also using any editor in combination with the Nextcloud sync client will work.

**Comment by korelstar** [comment]
I'm fine with that.

**Comment by stefan-niedermann** [comment]
I updated [my proposal](https://github.com/nextcloud/notes/issues/724#issuecomment-854476250). I think adding tools which explicitly mention Nextcloud Notes compatibility in their repositories is a good decision about whether or not they should be listed here (regardless whether or not they use the Notes API directly). I think this is also a chance to expand the ecosystem and maybe we can motivate people to use the Notes API directly in the future.  
I already opened two issues ([1](https://github.com/scharel/harbour-nextcloudnotes/issues/61), [2](https://github.com/mathiasconradt/nextcloud-notes-alexa-skill/issues/1)) to propose using version `1.x` instead of `0.2` of the API. Keeping track of those projects and listing them in the Notes wiki might also motivate further people to contribute here or there :slightly_smiling_face: :rocket:
